CMakeLists.txt

```diff
@@ -242,13 +288,8 @@ if (WIN32)
         /wd4800 # disable warning type' : forcing value to bool 'true' or 'false' (performance warning)
         /wd4503 # identifier' : decorated name length exceeded, name was truncated
     )
-    if(${ONNX_USE_MSVC_STATIC_RUNTIME})
```

Comment:
Why do we remove this? We should leave this as optional and update the CI if necessary.